alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+        description: Highest severity maximum query. The service will return instances whose highest severity is between *highestSeverityLevelMin* and *highestSeverityLevelMax*, inclusive.
+        example: 4
+      createdBy:
+        type: string
+        description: Created by query. The service will return instances whose *createdBy* fields are equal to the query, taking case into account.
+        example: TagRuleEngine
+      channel:
+        type: string
+        description: Channel query. The service will return instances whose *channel* fields are equal to the query, taking case into account. Supports wildcard match checking uses glob-style wildcards.
+        example: '*.System.Health.DiskSpaceUsePercentage'
+      properties:
+        type: object
+        description: Property query. The service will return instances whose *properties* fields contain all of the specified key/value pairs.
+        example:
+          key1: value1
+      keywords:
+        type: object
+        description: Keyword query. The service will return instances whose *keywords* fields contain all of the specified keywords.
+        example:
+          - keyword1
+          - keyword2
+  QueryRequest:
+    title: Query Request
+    allOf:
+    - $ref: '#/definitions/SubQuery'
+    - type: object
+      properties:
+        subQueries:
+          description: Additional queries. The service will return instances which match any of the queries in the subQueries array. If a top-level query was provided, the top-level query will be AND'ed with each of the sub queries.
+          type: array
+          items:
+            $ref: '#/definitions/SubQuery'
+        returnMostRecentlyOccurredOnly:
+          description: For the instances matched by this query, only return the most recent instance for each *alarmId*. In this context, recency is based on when the alarm instance occurred, not when it was last updated.
+          type: boolean
+        orderBy:
+          description: Whether or not results should be sorted in ascending or descending order relative to the alarm instance's *updatedAt* field.
+          type: string
+          enum: [DATE_UPDATED_FORWARD, DATE_UPDATED_BACKWARD]
+        skip:
+          description: Number of entries to skip in the response list. Typically used in combination with the take parameter to implement pagination.
+          type: integer
+          example: 0
+          default: 0
+        take:
+          description: Number of entries to include in the response list. Typically used in combination with the skip parameter to implement pagination.
+          type: integer
+          example: 10000
+          default: 10000
+          maximum: 10000
+
+responses:
+  Error:
+    description: Error
+    schema:
+      description: Error response
+      title: ErrorResponse
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+  QueryResponse:
+    description: Query response
```

Comment:
This description is meant to describe when the response is returned, and would typically be something like `OK` or `Success`.